cantools/database/can/formats/arxml/system_loader.py

```diff
@@ -1236,14 +1235,14 @@ def _load_multiplexed_pdu(self, pdu, frame_name, next_selector_idx):
             # specified indepently of that of the message. how should
             # this be handled?
 
-        if selector_signal.initial in selector_signal.choices:
-            selector_signal.initial = \
-                selector_signal.choices[selector_signal.initial]
+        if selector_signal.raw_initial in selector_signal.choices:
+            selector_signal.raw_initial = \
```

Comment:
once the choices are resolved, this becomes a physical (/scaled) value!? (i.e., it should be `scaled_initial`.) On that matter, I think the 'scaled_' prefix looks a bit weird for selector signals (but not a big deal IMO)...